cli/lib/tasks/download.js

```diff
@@ -79,6 +79,95 @@ const prettyDownloadErr = (err, version) => {
   return throwFormErrorText(errors.failedDownload)(msg)
 }
 
+/**
+ * Checks checksum and file size for the given file. Allows both
+ * values or just one of them to be checked.
+ */
+const verifyDownloadedFile = (filename, expectedSize, expectedChecksum) => {
+  if (expectedSize && expectedChecksum) {
+    debug('verifying checksum and file size')
+
+    return Promise.join(
+      util.getFileChecksum(filename),
+      util.getFileSize(filename)
+    ).spread((checksum, filesize) => {
```

Comment:
I think Promise.join without the function handler parameter is deprecated, per https://github.com/petkaantonov/bluebird/issues/1482

Reply:
thank you, I will update it to not use deprecated form